packages/ui/v2/core/banner.tsx

```diff
@@ -38,29 +35,28 @@ const Banner = (props: BannerProps) => {
         className
       )}
       {...rest}>
-      <div className={classNames("flex flex-row text-sm")}>
-        <div className="mr-3">{Icon && <Icon className="h-4 w-4" />}</div>
+      <div className={classNames("flex flex-col text-sm")}>
+        <div className="mb-2">{Icon && <Icon className="h-4 w-4" />}</div>
         <div className="flex flex-col space-y-2">
           <h1 className="font-semibold leading-none">{title}</h1>
           {description && <h2 className="font-normal leading-4">{description}</h2>}
           {props.variant === "error" && <p className="ml-4 pt-2 font-mono text-xs">{errorMessage}</p>}
         </div>
-      </div>
-      <div className="ml-auto self-start text-sm font-medium">
-        {props.actionText && (
-          <Button color="minimal" className={buttonStyle} onClick={() => props.onAction}>
-            Action
-          </Button>
-        )}
-        <Button
-          color="minimal"
-          className={buttonStyle}
-          onClick={(e) => {
-            setShow(false);
-            props.onDismiss(e);
-          }}>
-          Dismiss
-        </Button>
+        <div className="mt-4 flex w-full items-center gap-4 self-start text-sm font-medium">
+          {props.actionText && (
+            <button className="text-sm font-medium leading-4" onClick={() => props.onAction}>
```

Comment:
is there a reason why we changed Button for button?

Reply:
![image](https://user-images.githubusercontent.com/53316345/202648359-f3d500a1-34de-4224-b85f-eda60185c048.png)

left one is what the ui we want. Button in /components has py and px and background change on hover which we don't want so it would require over ridding a lot of styles if we use Button or should i just add another variant prop and use Button ?